src/Service/PaymentsService.php

```diff
@@ -79,12 +79,16 @@ public function createPreauthorizedPayment(PreauthorizedPayment $payment)
 	public function refundPayment($id, $amount, $items = NULL, $eet = NULL)
 	{
 		// without EET
-		if (is_null($items) || is_null($eet)) {
+		if ($items === NULL || $eet === NULL) {
 			return $this->makeRequest('POST', 'payments/payment/' . $id . '/refund', ['amount' => round($amount * 100)], Http::CONTENT_FORM);
 		}
 
 		// with EET
-		$data = array_merge(['amount' => round($amount * 100)], ['items' => $items], ['eet' => $eet]);
+		$data = array_merge(
+			['amount' => round($amount * 100)],
+			['items' => $items],
+			['eet' => $eet]
```

Comment:
Problém je, že to není pole, ale argumenty funkce array_merge.
Když přidám čárku za poslední element, spadnou testy na ParseError: syntax error, unexpected ')'